test/integration-test/commands/zlogout/logout.test.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, test } from '@oclif/test'
```

Comment:
is there a typo in the path to that file? It is `commands/zlogout/logout.test.ts` and I think the `z` shouldn't be there, right?

Reply:
We need the tests to run in a specific order and the logout should be the last test to run same as the login it should be the first test to run.

Reply:
Ah, yes that makes sense then. So all good.